### PR TITLE
fix: Add missing elements to package description JSON, fix some async issues.

### PR DESCRIPTION
**Pull request by paulhazen**
These changes represent a variety of changes that were needed in order to create a package from the project that can be subsequently added to a clean project via the Package Manager.

### CODE REVIEW

File: etc/PackageConfigurations/eos_package_description.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
         {"src":"Assets/Plugins/Source/Core/*.asmdef.meta", "dest": "Runtime/Core/"},
         {"src":"Assets/Plugins/Source/Core/*.cs", "dest": "Runtime/Core/"},
         {"src":"Assets/Plugins/Source/Core/*.cs.meta", "dest": "Runtime/Core/"},
-        
+        {"src":"Assets/Plugins/Source/Core/Utility/*.cs", "dest": "Runtime/Core/Utility/"},
+        {"src":"Assets/Plugins/Source/Core/Utility/*.cs.meta", "dest": "Runtime/Core/Utility/"},
```
Comment (arthur740212) [inline etc/PackageConfigurations/eos_package_description.json:39]: The scripts in `Source/Editor/Windows` are missing, I believe those are editor windows for users.

**Review by arthur740212 — CHANGES_REQUESTED** [review]
Other than the config.
This change might break the modular build tool.
But we might be able to fix that later.

**Review by andrew-hirata-playeveryware — APPROVED** [review]

File: etc/PackageConfigurations/eos_package_description.json (hunk 1 above)
Comment (paulhazen) [reply etc/PackageConfigurations/eos_package_description.json:39]: Excellent catch, thanks Arthur. I've since fixed the issue.

Comment (paulhazen) [reply etc/PackageConfigurations/eos_package_description.json:39]: For some reason, GitHub is not marking this file as being outdated 🤔

Comment (paulhazen) [reply etc/PackageConfigurations/eos_package_description.json:39]: [This link](https://github.com/PlayEveryWare/eos_plugin_for_unity/pull/579/commits/93d3c17fc148fbb000677d952efa16a141a194c3) to the commit shows the changes properly.

**Review by arthur740212 — APPROVED** [review]

**Pull request merged**